opendatahub-io/odh-dashboard
# Revamp empty state in Model Serving Global

## Description


Closes #1091 

## How Has This Been Tested?



1. Delete all Model Servers in all projects
2. Navigate to the `Model Serving` global page
3. Check that the empty state is changed

<img width="1430" alt="Screenshot 2023-09-18 at 15 57 21" src="https://github.com/opendatahub-io/odh-dashboard/assets/16117276/ac5e779a-3fbc-4a43-bd8a-02a6c9cb933a">


## Test Impact


Added coverage for empty state

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
